.github/workflows/auto-docs.yml

```
@@ -0,0 +1,426 @@
+        # Run documentation generator (using refactored system)
+        python -c "
+        import asyncio
+        import sys
+        from pathlib import Path
+        sys.path.append('.')
+        
+        async def generate_all_docs():
+            try:
+                # Use refactored DocumentationOrchestrator
+                from src.automation.documentation_orchestrator import DocumentationOrchestrator
+                
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)